packages/ns-api/README.md

```diff
@@ -6069,6 +6078,68 @@ Response example:
 {"message": "success"}
 ```
 
+### dns-list-blocked
+
+List blocked domains from the local DNS block list:
+```
+api-cli ns.threatshield dns-list-blocked
+```
+
+Response example:
+```json
+{
+  "data": [
+    {
+      "address": "dangerousdomain.com",
+      "description": "Lorem ipsum dolor sit amet"
+    },
+    {
+      "address": "nastydomain.net"
+    }
+  ]
+}
+```
+
+### dns-add-blocked
+
+Add a domain to the local DNS block list:
+```
+api-cli ns.threatshield dns-add-blocked --data '{"address": "nastydomain.net", "description": "my block1"}'
+```
+
+Response example:
+```json
+{"message": "success"}
+```
+
+It can raise the following validation errors:
+- `address_already_present` if the address is already inside the block list
+
+### dns-edit-blocked
+
+Change the description of an address already inside the local DNS block list:
+```
+api-cli ns.threatshield dns-edit-blocked --data '{"address": "nastydomain.net", "description": "My new desc"}'
+```
+
+It can raise the following validation errors:
+- `address_not_found` if the address is not inside the block list
+
+### dns-delete-blocked
+
+Delete an address from the local DNS block list:
+```
+api-cli ns.threatshield dns-delete-blocked --data '{"address": "nastydomain.net"}'
```

Comment:
Usually, other APIs always return all fields even if empty
```suggestion
api-cli ns.threatshield dns-delete-blocked --data '{"address": "nastydomain.net", "description": ""}'
```

Reply:
This is `dns-delete-blocked`, did you mean to change this on `dns-list-blocked`?

Reply:
yup sorry :(